docs/release-notes/RELEASE-NOTES-0.11.0.md

```diff
@@ -14,6 +14,10 @@ This release introduces backward incompatible changes in the chain format:
 * After upgrading your node, you will not have your previous balance (even if you keep your key pair);
 * Please ensure that you do not reuse a persisted blockchain produced by the previous releases "v0.10.x".
 
+Configuration related changes for this release (epoch.yaml):
+* The (`http` > `external` > `peer_address`) option has been removed, only the port option (`http` > `external` > `port`) is required if it differs from the internal port.
```

Comment:
I suggest removing this line as peer_address was already removed from user doc in https://github.com/aeternity/epoch/commit/8d86fe6da8f13f4cb0004df93f0770a47c0f455c#diff-76e731333fb756df3bff5ddb3b731c46